packages/beeq/src/components/slider/bq-slider.tsx

```diff
@@ -349,6 +374,19 @@ export class BqSlider {
             {this.maxRangeValue.toFixed(this.stepDecimalNumber)}
           </span>
         )}
+        {!this.isSingleSlider && this.enableTooltip && (
+          <bq-tooltip
+            class="absolute"
+            exportparts="base,trigger,panel"
+            alwaysVisible={this.isTooltipAlwaysVisible}
+            visible
+            distance={16}
+            style={{ left: `${this.getMaxRangeValue()}%`, fontVariant: 'tabular-nums' }}
+          >
+            <div class="absolute z-50 h-1 w-1" slot="trigger" />
+            {this.getMaxRangeValue()}
+          </bq-tooltip>
+        )}
```

Comment:
We can avoid duplication by creating separating render methods for those elements that are repeated. For example:

```ts
  private renderInput = ({
    value,
    onInputCallback,
    ref,
    ariaLabel,
  }: {
    value: number;
    onInputCallback: () => void;
    ref: (input: HTMLInputElement) => void;
    ariaLabel: string;
  }): HTMLInputElement => (
    <input
      class="bq-slider__input"
      type="range"
      min={this.min}
      max={this.max}
      step={this.step}
      value={String(value)}
      disabled={this.disabled}
      ref={ref}
      onInput={onInputCallback}
      onBlur={this.handleRangeInputBlur}
      onFocus={this.handleRangeInputFocus}
      aria-label={ariaLabel}
    />
  );

  private renderTooltip = (value: number): HTMLBqTooltipElement => (
    <bq-tooltip
      class="absolute"
      exportparts="base,trigger,panel"
      alwaysVisible={this.isTooltipAlwaysVisible}
      visible
      distance={16}
      style={{ left: `${value}%`, fontVariant: 'tabular-nums' }}
    >
      <div class="absolute z-50 h-1 w-1" slot="trigger" />
      {value}
    </bq-tooltip>
  );
```

and then we can call them as follow:

```ts
  render() {
    return (
      ...
        <div class="bq-slider__container">
          {/* SINGLE */}
          {this.renderInput({
            value: Number(this.getMinRangeValue()),
            onInputCallback: this.handleMinRangeInput,
            ref: (input: HTMLInputElement) => (this.minRangeInputElement = input),
            ariaLabel: 'Min Range',
          })}
          {this.enableTooltip && this.renderTooltip(Number(this.getMinRangeValue()))}
          {/* RANGE */}
          {!this.isSingleSlider &&
            this.renderInput({
              value: Number(this.getMaxRangeValue()),
              onInputCallback: this.handleMaxRangeInput,
              ref: (input: HTMLInputElement) => (this.maxRangeInputElement = input),
              ariaLabel: 'Max Range',
            })}
          {!this.isSingleSlider && this.enableTooltip && this.renderTooltip(Number(this.getMaxRangeValue()))}
          <div class="progress" ref={(div: HTMLDivElement) => (this.progressDivElement = div)}></div>
        </div>
        ...
```